src/sql/workbench/contrib/notebook/browser/notebookViews/notebookViewsGrid.component.html

```diff
@@ -0,0 +1,15 @@
+<!--
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+-->
+<div class="grid-stack">
```

Comment:
Since this is being used for notebooks specifically I'd suggest something like `nb-grid-stack` instead

Reply:
That makes sense, although the class `grid-stack` is required by the library for the default styles, so I added `nb-grid-stack` instead of replacing it. `nb-grid-stack` is now used for any styles specific to our use in notebooks.